db.js

```diff
@@ -1,6 +1,6 @@
 const asanas = [
   {
-    id: 1,
+    id: "1",
```

Comment:
Better not use id's like 1,2,3,... but a string with random numbers and letters. E.g. you could use nanoid()